packages/components/empty-state/src/MissingContent/MissingContent.mdx

```diff
@@ -0,0 +1,73 @@
+---
+title: 'MissingContent'
+type: 'component'
+slug: /empty-state/missingcontent/
+section: 'utils'
+typescript: ./MissingContent.tsx
+---
+
+`MissingContent` component can be used to render a placeholder for an empty table cell or any inline content that is missing. Its goal is to provide a system information without cluttering the view.
+
+## Import
+
+```jsx static=true
+import { MissingContent } from '@contentful/f36-empty-state';
+```
+
+## Example
+
+### Basic usage
+
+```jsx
+<Table>
+  <Table.Head>
+    <Table.Row>
+      <Table.Cell>Filename</Table.Cell>
+      <Table.Cell>Dimensions</Table.Cell>
+    </Table.Row>
+  </Table.Head>
+  <Table.Body>
+    <Table.Row>
+      <Table.Cell>Pony.jpg</Table.Cell>
+      <Table.Cell>160px x 300px</Table.Cell>
+    </Table.Row>
+    <Table.Row>
+      <Table.Cell>Cat.jpg</Table.Cell>
+      <Table.Cell>
+        <MissingContent label="No Dimensions available" />
+      </Table.Cell>
+    </Table.Row>
+    <Table.Row>
+      <Table.Cell>Dog.jpg</Table.Cell>
+      <Table.Cell>
+        <MissingContent label="No Dimensions available" />
+      </Table.Cell>
+    </Table.Row>
+  </Table.Body>
+</Table>
```

Comment:
Also on the new components we use separate folder/file examples, take a look into the other components structure